tests/v2/unit/adapters/torch/mmengine/mmdet/modules/models/detectors/conftest.py

```diff
@@ -81,3 +81,157 @@ def fxt_cfg_custom_atss(num_classes: int = 3) -> dict:
         train_cfg=train_cfg,
     )
     return cfg
+
+
+@pytest.fixture
+def fxt_cfg_custom_maskrcnn(num_classes: int = 3) -> dict:
```

Comment:
The decorator already indicates that this is a fixture. would it be an idea to use a more descriptive name to improve the readability? Something like:

`custom_mask_rcnn_config`?